src/components/atoms/LogoType/LogoType.tsx

```diff
@@ -0,0 +1,27 @@
+import styles from './LogoType.module.css'
+import Image from 'next/image'
+import AmberDefault from '@/../public/logotypes/amber-default.svg'
+import AmberChristmas from '@/../public/logotypes/amber-christmas.svg'
+
+export default function SeasonalLogoType() {
+    const month = new Date().getMonth();
+    if (month === 11 || month === 0) {
+        return <Image
+            src={AmberChristmas}
+            alt="Amber logo christmas edition"
+            style={{
+                position: 'relative',
+                width: 'auto',
+                // This image is 150% of the default logo
+                height: '150%',
+                top: '-50%',
```

Comment:
The text on christmas logo is 50% smaller, because the hat overflows from the text bounding box. This scales back the picture to 150% of size making the text on the logo appear in the same size as the default logo.

I use percents here so that I can bubble up the control over the actual size to the parent container